styles.css

```diff
@@ -1,552 +1,623 @@
+  font-weight: 500;
+}
+
+.text-regular {
+  font-weight: 400;
+}
+
+.text-light-grey {
+  color: #7a869a;
+}
+
+ul.project-language {
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-bottom: 1.3em;
+}
+
+ul.project-language li {
+  display: inline;
+  background-color: #ebebff;
+  margin-right: 0.5em;
+  padding: 0 1em;
+  border-radius: 0.5em;
+}
+
+.text-white {
+  color: #fff;
+}
+
+.contact-me {
+  background-color: #fff;
+}
+
+.contact-me .content-background {
+  background-image: url('images/contact_me_mobile.svg');
+  background-repeat: no-repeat;
+  background-position: center right;
+  background-size: contain;
+  height: 100%;
+  display: flex;
+  background-color: #6070ff;
+  border-top-left-radius: 5em;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact-me .content-background .content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.popup-card .card-title .h1 {
+  margin: 0.4em 0 0;
+}
+
+.action-buttons .button-outlined {
+  font-size: 1rem;
+}
+
+.contact-me .content-background .content .h1 {
+  margin-top: 2.6em;
+  margin-bottom: 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 80%;
+}
+
+.form-textarea {
+  resize: none;
+  height: 9em;
+}
+
+.button-outlined:hover {
+  color: #fff;
+  background-color: #6070ff;
+}
+
+.button-outlined:active {
+  color: #fff;
+  background-color: #2230d2;
+}
+
+.button-outlined:disabled {
+  border-color: #c1c7d0;
+  color: #5e6c84;
+  background-color: #fff;
+}
+
+.input {
+  font-size: 1.1rem;
+  padding: 0.7em;
+  margin-top: 1.3em;
+  border: 0.1em solid #6070ff;
+  border-radius: 0.75em;
+  min-height: 1.6em;
+  font-family: 'Poppins', sans-serif;
+}
+
+.textarea {
+  resize: none;
+  min-height: 10em;
+}
+
+.form .button-outlined {
+  margin-top: 1.3em;
+  align-self: flex-start;
+}
+
+.form .button-outlined:hover {
+  border: 1px solid #fff;
+}
+
+.form .button-outlined:active {
+  border: 1px solid #fff;
+}
+
+.contact-me .content-background .content .description {
+  display: flex;
+  justify-content: center;
+  font-size: 1.1em;
+  text-align: center;
+  max-width: 80%;
+}
+
+.popup-window {
+  visibility: hidden;
+  background-color: rgba(93, 99, 108, 0.5);
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(5px);
+}
+
+.popup-card {
+  width: 60%;
+  background-color: #fff;
+  border-radius: 1em;
+  padding: 0 1em;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.popup-card .action-buttons {
+  display: flex;
+  justify-content: space-around;
+}
+
+.popup-card .fa-solid.fa-xmark {
+  color: #67798e;
+  position: absolute;
+  top: 0.1em;
+  right: 0.1em;
+}
+
+.popup-card img {
+  align-self: center;
+}
+
+.popup-card .card-content {
+  margin-top: 1em;
+}
+
```

Comment:
- [ ] Good job so far, but I think you still need to work on your design. On the small screen, your popup card is a little broken the minimum mobile screen size is `375px`. Kindly fix that and make sure that your design is good on small screens like `375px` minimum 🙃 😉 

Your design

<img width="291" alt="Screenshot 2023-04-11 at 18 17 47" src="https://user-images.githubusercontent.com/74501523/231213926-564811d4-9ebd-4329-b4b8-45059ce1e895.png">

Figma design

<img width="305" alt="Screenshot 2023-04-11 at 18 18 26" src="https://user-images.githubusercontent.com/74501523/231214044-d03550e9-178b-454c-b4f7-f985049967be.png">